Source/Shared/arcana/functional/inplace_function.h

```diff
@@ -33,24 +33,38 @@
 namespace stdext
 {
     constexpr size_t InplaceFunctionDefaultCapacity = 32;
-
-    enum class inplace_function_operation
+        
+    template<bool SupportsCopy>
+    struct inplace_function_operation
     {
-        Destroy,
-        Copy,
-        Move
+        enum class operations_enum
+        {
+            Destroy,
+            Copy,
+            Move
+        };
+    };
+    template<>
+    struct inplace_function_operation<false>
+    {
+        enum class operations_enum
+        {
+            Destroy,
+            Move
+        };
     };
 
     template<typename SignatureT,
              size_t Capacity = InplaceFunctionDefaultCapacity,
-             size_t Alignment = alignof(std::max_align_t)>
+             size_t Alignment = alignof(std::max_align_t),
+             bool Copyable = true>
     class /*alignas(Alignment)*/ inplace_function;
 
-    template<typename RetT, typename... ArgsT, size_t Capacity, size_t Alignment>
-    class /*alignas(Alignment)*/ inplace_function<RetT(ArgsT...), Capacity, Alignment>
+    template<typename RetT, typename... ArgsT, size_t Capacity, size_t Alignment, bool Copyable>
```

Comment:
> bool Copyable

Should this default to true also? Technically this is a breaking change if we don't.

Reply:
This does default to true (unless I'm very misguided as to how this works). This is just a specialization of the templated class declared immediately above it, so it consumes the defaults provided by that declaration. In fact, I think this is the only specialization, so everywhere that we use this class ends up in this implementation with those defaults.

Reply:
makes sense.